docs/extensions/IconMarkupExtensions.md

```diff
@@ -0,0 +1,126 @@
+
+## Syntax
+
+**XAML**
+
+```xml
+<SwipeItems Mode="Reveal">
+
+    <!--Before-->
+    <SwipeItem Text="Accept">
+        <SwipeItem.IconSource>
+            <FontIconSource Glyph="&#xE10B;"/>
+        </SwipeItem.IconSource>
+    </SwipeItem>
+    
+    <!--After-->
+    <SwipeItem Text="Accept" IconSource="{ex:FontIconSource Glyph=&#xE10B;}"/>
+</SwipeItems>
+```
+
+## Properties
+
+| Property | Type | Description |
+| -- | -- | -- |
+| Glyph | string | The `string` representing the icon to display. |
+| FontSize | double | The size of the icon to display. |
+| FontFamily | FontFamily | The font family to use to display the icon. If `null`, "Segoe MDL2 Assets" will be used. |
+| FontWeight | FontWeight | The thickness of the icon glyph. |
+| FontStyle | FontStyle | The font style for the icon glyph. |
+| IsTextScaleFactorEnabled | bool | Indicates whether automatic text enlargement, to reflect the system text size setting, is enabled. |
+| MirroredWhenRightToLeft | bool | Indicates whether the icon is mirrored when the flow direction is right to left. |
+
+# BitmapIcon markup extension
+The [BitmapIcon markup extension](https://docs.microsoft.com/en-us/dotnet/api/microsoft.toolkit.uwp.ui.extensions.bitmapiconextension) is similar in structure to the two previous extensions, but it produces `BitmapIcon` instances instead of font-based icons.
+
+## Syntax
+
+**XAML**
+
+```xml
+<MenuFlyout>
+
+    <!--Before-->
+    <MenuFlyoutItem Text="Click me!">
+        <MenuFlyoutItem.Icon>
+            <BitmapIcon Source="/Assets/myicon.png"/>
+        </MenuFlyoutItem.Icon>
+    </MenuFlyoutItem>
+
+    <!--After-->
+    <MenuFlyoutItem
+        Text="No, click me!"
+        Icon="{ex:BitmapIcon Source=/Assets/myicon.png}" />
+</MenuFlyout>
+```
+
+## Properties
+
+| Property | Type | Description |
+| -- | -- | -- |
+| Source | Uri | The `Uri` representing the image to display. |
+| ShowAsMonochrome | bool | Indicates whether to display the icon as monochrome. |
+
+## Requirements
+
+| Device family | Universal, 10.0.16299.0 or higher   |
+| -- | -- |
+| Namespace | Microsoft.Toolkit.Uwp.UI.Extensions |
+| NuGet package | [Microsoft.Toolkit.Uwp.UI](https://www.nuget.org/packages/Microsoft.Toolkit.Uwp.UI/) |
+
+## API Source Code
+
+- [FontIconExtension source code](https://github.com/Microsoft/WindowsCommunityToolkit//blob/master/Microsoft.Toolkit.Uwp.UI/Extensions/Markup/FontIconExtension.cs)
+- [FontIconSourceExtension source code](https://github.com/Microsoft/WindowsCommunityToolkit//blob/master/Microsoft.Toolkit.Uwp.UI/Extensions/Markup/FontIconSourceExtension.cs)
+- [BitmapIconExtension source code](https://github.com/Microsoft/WindowsCommunityToolkit//blob/master/Microsoft.Toolkit.Uwp.UI/Extensions/Markup/BitmapIconExtension.cs)
+
+## Related Topics
+
+- [MarkupExtension Class](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.markup.markupextension)
```

Comment:
This is the base class, eh?

We should also provide links to all the UWP system docs for these types.

Reply:
E.g. 

[FontIcon API](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.FontIcon)

Reply:
Fixed in https://github.com/MicrosoftDocs/WindowsCommunityToolkitDocs/pull/313/commits/1e23852a482a18b123ca3bd556c0969a8297315e.